nodetypes/radon.nodes.monitoring/PushGateway/NodeType.tosca

```diff
@@ -0,0 +1,96 @@
+tosca_definitions_version: tosca_simple_yaml_1_3
+
+node_types:
+  radon.nodes.monitoring.PushGateway:
+    derived_from: radon.nodes.docker.DockerApplication
+    metadata:
+      targetNamespace: "radon.nodes.monitoring.PushGateway"
+      abstract: "false"
+      final: "false"
+    properties:
+      user_email:
+        type: string
+        default: { get_property: [SELF, user_email] }
```

Comment:
Using `{ get_property: [SELF, user_email] }` on the `user_email` property could lead to recursion loop. 

Can we use `get_input` here?

Reply:
Done!